paddle/fluid/operators/fill_constant_op.cc

```diff
@@ -116,6 +116,15 @@ class FillConstantOpMaker : public framework::OpProtoAndCheckerMaker {
                   "memory. Otherwise, fill output variable to the running "
                   "device")
         .SetDefault(false);
+    AddAttr<int>("place_type",
+                 "(int, default 0) allow mamually setting place where the "
+                 "variable should be hold. "
+                 "0: not set manually, determine the place by executor. "
```

Comment:
Better set default to -1. Otherwise, the meaning of `place_type` in fill_constant is not consistent with the `dst_place_type` in MemcpyOp.

Reply:
updated